curriculum/challenges/english/25-front-end-development/workshop-hotel-feedback-form/66a95d0eff8168747805f1f3.md

```diff
@@ -32,7 +32,7 @@ Below your `label` element, add a checkbox input with the `id`, `name` and `valu
 You should have a `label` element below your `legend`.
 
 ```js
-assert(document.querySelector("fieldset:nth-of-type(3) legend + label"));
+assert.strictEqual(document.querySelector("fieldset:nth-of-type(3) legend + label"));
```

Comment:
Here is the step that is failing the test.
This is the incorrect use of `strictEqual ` 

https://www.chaijs.com/api/assert/#method_strictequal

Right now, you are not comparing two values. So `strictEqual ` won't work here.

`assert.exists` is the better option

https://www.chaijs.com/api/assert/#method_exists

Reply:
I see, thanks for the guidance Jessica! @jdwilkin4